tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -2577,7 +2577,7 @@ def create_full_like_arg_names(val):
         a = full_like(arr, val, float, 'F', shape = (2,3))
         return a[0,0],a[0,1],a[0,2],a[1,0],a[1,1],a[1,2]
 
-    size = randint(10)
+    size = uniform(10)
```

Comment:
Can you explain this change?

Reply:
same as above the type set in decorators is real but randint generate int